stylesheet.css

```diff
@@ -0,0 +1,275 @@
+    padding-top: 100px;
+    background-color: #d1e0ef;
+}
+.external_button:hover{
+    background-color: #fae3e5;
+}
+
+.external_button{
+    padding:18px;
+    background-color: #FFFFFF;
+    border-radius: 45px;
+    margin: 10px 10px 0 0;
+    display:inline-block;
+
+}
+/* My experience section styling */
+.my_experience{
+    background-color: #fae3e5;
+    margin: 0px;
+    display: grid;
+    grid-template-columns: 40% 30% 20%;
+}
+.cv_image{
+    max-width: 500px;
+}
+.cv_icon{
+    place-self: center;
+}
+.experience_text{
+    padding: 100px;
+}
+/* Contact form styling */
+.contact_form_section{
+    background-image: url("Images/Background.jpg");
+    height: 850px;
+    padding-top: 100px;
+}
+.contact_form{
+    background-color: #FFFFFF;
+    padding: 100px;
+    margin: auto;
+    width: 50%;
+}
+.contact_form p{
+    margin-left: 10px;
+}
+.contact_form h2{
+    margin: 10px;
+}
+input{
+    display: block;
+    padding: 10px;
+    margin: 10px;
+    background-color: #d1e0ef;
+    border: none;
+    width: 90%;
+}
+textarea{
+    margin: 0 10px;
+    padding: 1px 2px;
+    background-color: #d1e0ef;
+    border: none;
+    padding: 10px;
+    color: #000000;
+    width: 90%;
+}
+textarea::placeholder{
+    color: #6a6a6a;
+}
+/* Footer bar styling */
+footer{
+    height: 70px;
+    background-color: #fae3e5;
+    padding: 20px;
+}
+footer a{
+    padding: 5px;
+    text-decoration: none;
+}
+.email_link{
+    float: right;
+}
+a{
+    color:#214794;
+    text-decoration: none;
+}
+@media only screen and (max-width:1266px){
+    .my_experience{
+        grid-template-columns: 100%;
+        background-size: cover;
+        background-position: center;
+        margin: 0px;
+    }
+    .experience_text{
+        padding: 10px 0 10px 90px
+    }
+}
+@media only screen and (max-width: 998px){
+    .intro{
+        grid-template-columns: 100%;
+        margin:0;
+        padding:0;
+        background-color: #fae3e5;
+    }
+    .intro_item.intro_text{
+        background-color: #fae3e5;
+        padding: 10px ;
+        width: 100%;
+        margin: 0;
+    }
+    header{
+        background-image: none;
+        height:auto;
+    }
+    .header_button{
+        display:none;
+    }
+    img{
+        max-width: 100%;
+        height: auto;
+        margin-left: auto;
+        margin-right: auto;
+        
+    }
+    .project_image img{
+        height: auto;
+        width:auto;
+        display:block;
+        margin-right:auto;
+        margin-left:auto;
+    
+    }
+    .project_text{
+        margin: 0;
+    }
+    .my_work{
+        grid-template-columns: 100%;
+        margin: 0;
+        padding: 10px;
+    }
+    .my_work_two{
+        grid-template-columns: 100%;
+        margin: 0;
+        padding: 10px;
+    }
+    .contact_form{
+        padding: 10px;
+        width: 80%;
+    }
+}
+@media only screen and (max-width: 617px){
+    nav a{
+        display: grid;
+		grid-template-columns: 100%;
```

Comment:
Check your indent here

Reply:
I couldn't see this on my screen, let me know if it's still an issue